apps/web/src/studio/StudioPageLayout.tsx

```diff
@@ -1,9 +1,16 @@
 import { useStudioState } from './StudioStateProvider';
 import { PrivatePageLayout } from '../components/layout/components/PrivatePageLayout';
 import { LocalStudioPageLayout } from '../components/layout/components/LocalStudioPageLayout';
+import { ROUTES } from '../constants/routes';
+import { Outlet, useLocation } from 'react-router-dom';
 
 export function StudioPageLayout() {
   const state = useStudioState();
+  const { pathname } = useLocation();
+
+  if (pathname.startsWith(ROUTES.STUDIO_ONBOARDING)) {
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
☑ **chore:** Extract to helper with other `isStudioRoute` function to encourage people to prevent diverging implementations